core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -3149,7 +3154,9 @@ private boolean syncMetadata(JournalContext journalContext, LockedInodePath inod
         loadMetadataAndJournal(inodePath, LoadMetadataOptions.defaults().setCreateAncestors(true)
             .setLoadDescendantType(syncDescendantType), journalContext);
       } catch (Exception e) {
-        LOG.error("Failed to load metadata for path: {} error: {}", inodePath.getUri(),
+        // This may be expected. For example, when creating a new file, the UFS file is not
```

Comment:
Can we tell the expected case by checking certain state on the metadata?

Reply:
Hrmmm, not sure. If we can figure out a better way to determine if it is expected or not, we can also update the log message in `loadMetadataIfNotExistAndJournal`.

Reply:
okay, we have go with it for now